skimage/morphology/convex_hull.py

```diff
@@ -2,7 +2,8 @@
 from itertools import product
 import numpy as np
 from scipy.spatial import ConvexHull
-from ..measure.pnpoly import grid_points_in_poly
```

Comment:
Should we make changes to `grid_points_in_poly` instead?

Reply:
I'm not sure which functions use that method, but if they could benefit from this speedup for drawing polygons with many edges (though I can't say at how many edges you start seeing the speedup), applying the changes there instead could work well.